spec/serializers/user_summary_serializer_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+require_dependency 'user'
+
+describe UserSummarySerializer do
+
+  context "works" do
```

Comment:
I don't think it makes sense to have a context here. Contexts are only important for separating things.